File: app/src/main/java/com/philkes/pin2pdf/api/pinterest/model/ImagesResponse.java
package com.philkes.pin2pdf.api.pinterest.model;

import com.google.gson.annotations.SerializedName;

public class ImagesResponse {

    @SerializedName("236x")
    private ImageResponse imgSmall;

    @SerializedName("474x")
    private ImageResponse imgBig;

    public ImagesResponse() {
    }

    public ImageResponse getImgSmall() {
        return imgSmall;
    }

    public void setImgSmall(ImageResponse imgSmall) {
        this.imgSmall=imgSmall;
    }

    public ImageResponse getImgBig() {
        return imgBig;
    }

    public void setImgBig(ImageResponse imgBig) {
        this.imgBig=imgBig;
    }
}

File: app/src/main/java/com/philkes/pin2pdf/api/pinterest/model/ImagesResponse.kt
package com.philkes.pin2pdf.api.pinterest.model

import com.google.gson.annotations.SerializedName

class ImagesResponse {
    @SerializedName("237x")
    var imgSmall: ImageResponse? = null

    @SerializedName("564x")
    var imgBig: ImageResponse? = null
}